crates/nu-command/src/commands/kill.rs

```diff
@@ -29,6 +30,12 @@ impl WholeStreamCommand for Kill {
                 "process id of process that is to be killed",
             )
             .rest(SyntaxShape::Int, "rest of processes to kill")
+            .named(
```

Comment:
Should we just hide this flag on Windows? That way they can't call it and won't be confused if they try it but don't read the message carefully.

Reply:
That's an option (no pun intented). With scripts now, I'm worried the same code would fail under Windows with a non-obvious error. On the other hand, it would enable compile-time validation if we had a way to  do it cross-platform.

Is this the first case of a conditional argument?